docs/setup/release-notes.md

```diff
@@ -4,6 +4,117 @@
+* [X] **Sedona Spark & Flink** Added geodesic / geography functions:
+    * ST_DistanceSphere
+    * ST_DistanceSpheroid
+    * ST_AreaSpheroid
+    * ST_LengthSpheroid
+* [X] **Sedona Spark & Flink** Introduced `SedonaContext` to unify Sedona entry points.
+* [X] **Sedona Spark** Support Spark 3.4.
+* [X] **Sedona Spark** Added a number of new ST functions.
+* [X] **Zeppelin** Zeppelin helium plugin supports ploting geometries like linestring, polygon.
+
+### API change
+
+* **Sedona Spark & Flink** Introduced a new entry point called SedonaContext to unify all Sedona entry points in different compute engines and deprecate old Sedona register entry points. Users no longer have to register Sedona kryo serializer and import many tedious Python classes.
+    * **Sedona Spark**:
+        * Scala:
+        ```scala
+        import org.apache.sedona.spark.SedonaContext
+        val sedona = SedonaContext.create(SedonaContext.builder().master("local[*]").getOrCreate())
+        sedona.sql("SELECT ST_GeomFromWKT(XXX) FROM")
+        ```
+        * Python:
+        ```python
+        from sedona.spark import *
+
+        config = SedonaContext.builder().\
+           config('spark.jars.packages',
+               'org.apache.sedona:sedona-spark-shaded-3.0_2.12:1.4.1,'
+               'org.datasyslab:geotools-wrapper:1.4.0-28.2'). \
+           getOrCreate()
+        sedona = SedonaContext.create(config)
+        sedona.sql("SELECT ST_GeomFromWKT(XXX) FROM")
+        ```
+    * **Sedona Flink**:
+    ```java
+    import org.apache.sedona.flink.SedonaContext
+    StreamTableEnvironment sedona = SedonaContext.create(env, tableEnv);
+    sedona.sqlQuery("SELECT ST_GeomFromWKT(XXX) FROM")
+    ```
+
+### Bug
+
+<ul>
+<li>[<a href='https://issues.apache.org/jira/browse/SEDONA-266'>SEDONA-266</a>] -         RS_Values throws UnsupportedOperationException for shuffled point arrays
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/SEDONA-267'>SEDONA-267</a>] -         Cannot pip install apache-sedona 1.4.0 from source distribution
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/SEDONA-273'>SEDONA-273</a>] -         Set a upper bound for Shapely, Pandas and GeoPandas
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/SEDONA-277'>SEDONA-277</a>] -         Sedona spark artifacts for scala 2.13 do not have proper POMs
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/SEDONA-283'>SEDONA-283</a>] -         Artifacts were deployed twice when running mvn clean deploy
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/SEDONA-284'>SEDONA-284</a>] -         Property values in dependency deduced POMs for shaded modules were not substituted
+</li>
+</ul>
+
+### New Feature
+
+<ul>
+<li>[<a href='https://issues.apache.org/jira/browse/SEDONA-196'>SEDONA-196</a>] -         Add ST_Force3D to Sedona
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/SEDONA-239'>SEDONA-239</a>] -         Implement ST_NumPoints
+</li>
```

Comment:
@jiayuasu @Kontinuation ST_Translate implementation has also been merged to master, and should be included in the release notes.
JIRA link: https://issues.apache.org/jira/browse/SEDONA-302

Reply:
Addressed in https://github.com/apache/sedona/commit/b2db51abe7699f77ddf9cfbb5ca1fea7f4df6ccd